pkg/operation/operation.go

```diff
@@ -169,6 +169,11 @@ func (o *Operation) InitializeShootClients() error {
 	return nil
 }
 
+//ComputePrometheusIngressFQDN computes full qualified domain name for prometheus ingress sub-resource and returns it.
+func (o *Operation) ComputePrometheusIngressFQDN() string {
```

Comment:
Use the function [here](https://github.com/gardener/gardener/blob/098f1488a45b7aaa048a859acadd4e72df414d1d/pkg/operation/botanist/secrets.go#L79) as well.

Reply:
The ```GetIngressFQDN``` function does more, it adds as well ```s.Info.Spec.IngressDomain``` to the string.

Reply:
Oh, ok, not seen this. This was not what I meant. The `ComputePrometheusIngressFQDN` should only compute the Ingress FQDN for Prometheus. If you need the `https` protocol in your case in front of then add it there.

```golang
// ComputePrometheusIngressFQDN computes full qualified domain name for prometheus ingress sub-resource and returns it.
func (o *Operation) ComputePrometheusIngressFQDN() string {
    return o.Seed.GetIngressFQDN("p", o.Shoot.Info.Name, o.Garden.Project.Name)
}
```